packages/api-plugin-fulfillment-type-pickup/src/checkAndCreateFulfillmentType.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @summary Called on startup to create the root entry of this fulfillment type in Fulfillment collection
+ * @param {Object} context Startup context
+ * @param {String} shopId Shop ID
+ * @returns {Boolean} true if entry exist or insert success else false
+ */
+export default async function checkAndCreateFulfillmentType(context, shopId) {
+  const { collections: { Fulfillment } } = context;
+
+  const pickupRecord = await Fulfillment.findOne({ fulfillmentType: "pickup", shopId });
+  if (!pickupRecord) {
+    const groupInfo = {
+      name: "Pickup Provider",
+      shopId,
+      provider: {
+        enabled: true,
+        label: "Pickup",
+        name: "pickup"
+      },
+      fulfillmentType: "pickup"
+    };
+    await context.mutations.createFulfillmentType(context.getInternalContext(), groupInfo);
+  }
+  return true;
```

Comment:
short return first makes it looks better

```suggestion
  if (pickupRecord) return true;
  
    const groupInfo = {
      name: "Pickup Provider",
      shopId,
      provider: {
        enabled: true,
        label: "Pickup",
        name: "pickup"
      },
      fulfillmentType: "pickup"
    };
    await context.mutations.createFulfillmentType(context.getInternalContext(), groupInfo);
  return true;
```